students/KevinWaldron/lesson04/trigrams.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+import sys
+import random
+
+def read_text(file):
+    """
+    Reads the text from the given input text file into a list of words that were separated by whitespace
+    :param file: input text file
+    :returns: the list of words
+    """
+    text = []
+    with open(file, 'r') as f:
```

Comment:
avoid using `file` for var name (it is a built-in)

(notice different color syntax highlight)